packages/ssi-types/__tests__/dif_pe_examples/vp/vp_open_id_connect.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
since we don't support/use this, remove it

Reply:
removed and dir name changed